src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -512,6 +520,45 @@ The trusted code can store capabilities on the trusted stack when calling out of
 when returning to the same compartment.
 =====
 
+[#ycfg,reftext="ycfg"]
+===== CHERI Capability Encoding (ycfg)
+
+The <<ycfg>> register is used to identify which CHERI capability encoding is used by the platform.
+The capability encoding both determines the in-memory representation of a CHERI capability
+and entails a set of extensions present atop {cheri_base_ext_name}.
+The CSR exists in the _writable_ namespace and all bits are defined to be WARL,
+but, absent any future extensions, it is expected that each implementation has exactly one legal value.
+This CSR is always present in the machine mode read/write CSR space;
+if the hart supports user mode, a `uycfg` register exists,
+if the hart supports supervisor mode, an `sycfg` register exists, and
+if the hart supports hypervisor mode, a `vsycfg` register exists.
+`mycfg` is always read-only, as is `vsycfg`, if it exists.
+`uycfg` is a read-only alias of `sycfg`, if it exists, or of `mycfg` otherwise.
+If the hypervisor extension is enabled, then `sycfg` is WARL,
+updatable from M or H mode with <<pcc>> granting <<asr_perm>>.
+
+The base specification defines the following values for `mycfg`, `sycfg`, and `uycfg`:
+|===
+|Version value|Encoding
+|0|Reserved (unspecified capability encoding)
+|0x001|<<app_cap_description>> capabilities, without <<section_zylevels1>> features
+|0x101|<<app_cap_description>> capabilities, with <<section_zylevels1>> features
+|0x002|Reserved for CHERIoT's (RV32-only) encoding
+|===
+
+Future extensions that change capability encoding are expected to define new values.
+Extensions that define encodings derived from <<app_cap_description>>
+are encouraged to preserve the bottom 8 bits as `0x01`.
```

Comment:
Not encouraged. If it is the same base encoding, with the extension done in a compatible manner, then that *is* 0x01 and listing anything else is gratuitous.